### PR TITLE
Depend only on jdom2

### PR DESCRIPTION
**Pull request by eparejatobes**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -8,7 +8,7 @@ javaVersion   := "1.8"
 
 libraryDependencies ++= Seq(
   "bio4j"         % "angulillos"    % "0.4.1",
-  "ohnosequences" % "bioinfo-util"  % "1.5.0"
+  "org.jdom"      % "jdom2"          % "2.0.6"
 )
 
 dependencyOverrides ++= Set(
```
Comment (laughedelic) [inline build.sbt]: Have you tried to remove `dependencyOverrides`?

Comment (laughedelic) [inline build.sbt]: I see that commons-* come from http-client thing and jackson-* from blueprints.

Comment (eparejatobes) [inline build.sbt]: removed everything and compiles cleanly

**Comment by eparejatobes** [comment]
LGTM

**Pull request merged**